doc/SONiC_202006_Release_Notes.md

```diff
@@ -0,0 +1,134 @@
+
+#### Getting docker ready for Debian 10
+
+<br> Refer [HLD document]() and below mentioned PR's for more details. 
+<br> **Pull Requests** : [4671](https://github.com/Azure/sonic-buildimage/pull/4671), [4727](https://github.com/Azure/sonic-buildimage/pull/4727), [4726](https://github.com/Azure/sonic-buildimage/pull/4726), [4665](https://github.com/Azure/sonic-buildimage/pull/4665), [4515](https://github.com/Azure/sonic-buildimage/pull/4515),  [4598](https://github.com/Azure/sonic-buildimage/pull/4598), [4529](https://github.com/Azure/sonic-buildimage/pull/4529), [4480](https://github.com/Azure/sonic-buildimage/pull/4480)
+
+#### Port Mirroring
+This feature describes the high level design details on Port/Port-channel mirroring support, dynamic session management, ACL rules can continue to use port/ERSPAN sessions as the action, Configuration CLI for mirror session.
+<br> Refer [HLD document](https://github.com/Azure/SONiC/blob/e8c86d1b3a03d6320727ff148966081869461e4a/doc/SONiC_Port_Mirroring_HLD.md) and below mentioned PR's for more details. 
+<br> **Pull Requests** : [1314](https://github.com/Azure/sonic-swss/pull/1314), [936](https://github.com/Azure/sonic-utilities/pull/936) 
+
+#### Proxy ARP  
+When an interface is enabled with "proxy_arp", the same is enabled in the kernel. ASIC ARP packet action is also updated to trap these packets to CPU in those interfaces.
+<br> Refer [HLD Document](https://github.com/Azure/SONiC/blob/master/doc/arp/Proxy%20Arp.md) for more details. 
+<br> **Pull Requests** :  [617](https://github.com/Azure/SONiC/pull/617)
+
+#### Pytest 100% moved from ansible to Pytest 
+
+#### SPytest
+
+<br> Refer [HLD Document]() for more details. 
+<br> **Pull Requests** :  [1533](https://github.com/Azure/sonic-mgmt/pull/1533)
+
+#### Thermal control 
+Thermal control daemon has been added to monitor the temperature of devices (CPU, ASIC, optical modules, etc) and the running status of fan. It retrieves the switch device temperatures via platform APIs and raises alarms when the high/low thresholds are hit.It also stores temperature values fetched from sensors and thermal device running status to the DB.
+<br> Refer  [HLD Document](https://github.com/Azure/SONiC/blob/master/thermal-control-design.md) for more details.
+<br> **Pull Requests** :  [73](https://github.com/Azure/sonic-platform-common/pull/73), [777](https://github.com/Azure/sonic-utilities/pull/777), [49](https://github.com/Azure/sonic-platform-daemons/pull/49), [3949](https://github.com/Azure/sonic-buildimage/pull/3949),[832](https://github.com/Azure/sonic-utilities/pull/832) 
+
+
+<br>
+
```

Comment:
Addressed it. Pls check and confirm.

Reply:
@kannankvs instead of mixing them into thermal control, I more incline adding two addtional features - "PSU and FAN LED management" and "PSU, thermal and FAN plugin extension".

"PSU and FAN LED management" - The PSU and FAN LED on switch will be set according to PSU and FAN presence and running status, for example if there is a failure happening to PSU or FAN, the corresponding LED will be set to red. 
On new plugins for fan, thermal and PSU the PSU plugin was extended with voltage, current and power supported, and the fan and thermal plugins were introduced.

Reply:
Changed. Pls check and confirm.